pagure_exporter/conf/standard.py

```diff
@@ -117,10 +117,10 @@
 destcloc = "UNAVAILABLE"
 
 # Regular expression to help detection of the GitLab Runners token in response body
-detect = "\"runners_token\":\"[A-Za-z0-9]+\""
+detect = r"\"runners_token\":\"([^\"]*)\""
```

Comment:
If only replacing it here, I'd probably throw in a `\s*` just in case there are spaces.

```suggestion
detect = r"\"runners_token\":\s*\"([^\"]*)\""
```

Reply:
But if you wanted this to be:

```suggestion
detect = r"\b(?:glrt-|GR1348941)[\w\-]{20,23}"
```

It should find tokens regardless where they are.

And cutout could just become:

```python
cutout = "EXPUNGED"
```

Reply:
I generated some GitLab Runner tokens to see if the pattern qualifies and the most recent comment fits the bill perfectly. I am going to set the mentioned filter, reset the recorded cassettes and run the tests again to see if the expungement works fine before pushing the changes back again.